src/ghdcbot/adapters/github/rest.py

```
@@ -0,0 +1,1145 @@
+            try:
+                response = self._client.get(
+                    f"/repos/{owner}/{repo}/issues/{issue_number}",
+                    headers={"Accept": "application/vnd.github+json"},
+                )
+                if response.status_code == 200:
+                    issue = response.json()
+                    author = (issue.get("user") or {}).get("login")
+                    if author:
+                        issue_authors[issue_number] = author
+            except Exception:  # noqa: BLE001
+                pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)